### ISSUE TITLE
I have a question about heatmap

### ISSUE THREAD

**Issue by Jaekyu-Sim** [michalfaber/keras_Realtime_Multi-Person_Pose_Estimation#80]
I'm very thanksful about your github page.

When I see your github page about Pose_Estimation project,

I saw that we need heatmap 19(because 18 joint + 1 background)

but I can understand why I have to consider background

**Comment by michalfaber** [comment]
@Jaekyu-Sim @USTClj I’ve trained the model without background channel and it looks like it doesn’t affect the detections. The loss values are slightly smaller. I thought that the background could be helpful in calculating coordinates - non maximum suppression around the lines: 
all_peaks = []
peak_counter = 0
for part in range(19-1):
…but authors of original solution don’t use the background channel - maybe it was a result of some previous experiments. I noticed that when I add the background I get twice as many peaks - most of them don’t affect the final result and can be discarded.
